agents/lakeshore372/LS372_agent.py

```diff
@@ -977,6 +985,79 @@ def get_still_output(self, session, params=None):
 
         return True, "Current still output is {}".format(still_output)
 
+    def input_configfile(self, configfile, filepath='/ls372configs/'):
+        """ Upload 372 configuration file to initialize channel/device
+        settings upon Agent startup.
+
+        Args:
+            configfile (str): name of .yaml config file
+        """
+        # include /ls372configs/ dir where configfiles are in docker container
+        ls372config = os.path.join(filepath, configfile)
```

Comment:
To use `$OCS_CONFIG_DIR` here you'll need to access the environment variable via `os.environ.get("OCS_CONFIG_DIR", "/config/")`. (`OCS_CONFIG_DIR` is set in the containers to `/config` by default, but this default to that in the rare case it isn't set.)